internal/cmd/schema-registry/command_exporter_create.go

```diff
@@ -42,7 +42,9 @@ func (c *command) newExporterCreateCommand() *cobra.Command {
 	pcmd.AddEnvironmentFlag(cmd, c.AuthenticatedCLICommand)
 	pcmd.AddOutputFlag(cmd)
 
-	_ = cmd.MarkFlagRequired("config-file")
+	cobra.CheckErr(cmd.MarkFlagFilename("config-file"))
```

Comment:
@Clarence97 Is `--config-file` always a JSON file?

Reply:
No, it's specified as lines of key/value pairs. https://github.com/confluentinc/cli/blob/main/internal/pkg/properties/properties.go#L36

Reply:
Its description is "Exporter config file." Maybe you're confusing it with another flag?

Reply:
Ah, right. Thank you!